src/_tests/fixtures/43695/result.json

```diff
@@ -1,6 +1,6 @@
 {
   "pr_number": 43695,
-  "targetColumn": "Waiting for Code Reviews",
+  "targetColumn": "Needs Maintainer Review",
```

Comment:
The tests don't include the time when the fixture was taken yet, so over time these tests go from "You + other DT module owners" to "DT maintainers"